site/app/models/gradeable/GradedComponent.php

```diff
@@ -55,6 +55,10 @@ class GradedComponent extends AbstractModel {
     protected $graded_version = 0;
     /** @property @var \DateTime The time which this grade was most recently updated */
     protected $grade_time = null;
+    /** @property @var string The Id of the verifier who verified the grade */
+    protected $verifier_id = "";
```

Comment:
To keep this parallel with `grader`, I think we want to keep a verifier user object instead of just an id string.